src/data.js

```diff
@@ -15682,6 +15682,55 @@ module.exports = [
       'Ruby on Rails',
     ],
   },
+  {
+    name: 'lime360',
+    description: 'idk',
+    url: 'https://lime360.neocities.org/uses',
+    emoji: '🍋',
+    country: '🇺🇦',
+    computer: 'windows',
+    phone: 'androis',
```

Comment:
"phone" must be one of [iphone, android, windowsphone, flipphone]

Reply:
oh sorry I misspelled android